tests/UnimodMapper_test.py

```diff
@@ -158,3 +158,17 @@ def test_conversion(self, conversion):
         #         with self.assertRaises(SystemExit) as system_exit_check:
         #             self.alt_mapper._parseXML()
         #         self.assertEqual(system_exit_check.exception.code, 1)
+
+    def test_write(self):
+        xml_file = os.path.join(os.path.dirname(__file__), "test_only_unimod.xml")
+        assert os.path.exists(xml_file) is False
```

Comment:
What do you assert for? I would simply move it to the bottom to assert, that the `os.remove()` did its job. Logic wise, even if the file would exist the dict will be appended, which I believe is desired.

Reply:
I don't want to overwrite an existing "test_only_unimod.xml" if a user name it that way.